addon/components/apex-chart.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@glimmer/component';
+import ApexCharts from 'apexcharts';
+import { guidFor } from '@ember/object/internals';
+import { action } from '@ember/object';
+import { isNone } from '@ember/utils';
+
+export default class ApexChart extends Component {
+  get className() {
+    return guidFor(this);
+  }
+
+  @action
+  destroyChart() {
+    if(this.chart) this.chart.destroy();
+  }
+
+  @action
+  createChart(element) {
+    if(isNone(this.args.chartOptions)){
+      return;
+    }
+
+    if(this.chart) {
+      this.destroyChart();
+    }
```

Comment:
For the case where you update to an empty chart:
```suggestion
    this.destroyChart();

    if(isNone(this.args.chartOptions)){
      return;
    }

```